zerver/tests/test_realm_filters.py

```diff
@@ -27,7 +27,7 @@ def test_create(self) -> None:
         self.login(email)
         data = {"pattern": "", "url_format_string": "https://realm.com/my_realm_filter/%(id)s"}
         result = self.client_post("/json/realm/filters", info=data)
-        self.assert_json_error(result, 'This field cannot be blank.')
+        self.assert_all_json_errors(result, ['This field cannot be blank.', 'No groups found in URL format string.'])
```

Comment:
Does the frontend handle multiple errors?

Reply:
Yes.

---

![image](https://user-images.githubusercontent.com/8033238/73116989-3adc8a00-3f65-11ea-993d-c05968794382.png)


This is the result with the API returning:

```json
{
    "result":"error",
    "msg":"This field cannot be blank.",
    "errors":{
        "pattern":[
            "This field cannot be blank."
        ],
        "url_format_string":[
            "This field cannot be blank."
        ],
        "__all__":[
            "No groups found in URL format string."
        ]
    }
}
```